scripts/everything/install-context.reg

```diff
@@ -1,22 +1,22 @@
 Windows Registry Editor Version 5.00
 
 [HKEY_CURRENT_USER\Software\Classes\Directory\shell\Everything]
-@="search with &Everything"
```

Comment:
The `&` are to provide an Alt key shortcut. Why remove it?

Reply:
I wasn't aware and thought that it was a mistake, I'll fix that, sorry.

Reply:
> The `&` are to provide an Alt key shortcut. Why remove it?

Fixed